pkg/virt-handler/vm.go

```diff
@@ -2812,7 +2812,7 @@ func (c *VirtualMachineController) vmUpdateHelperMigrationTarget(origVMI *v1.Vir
 	}
 
 	// configure network inside virt-launcher compute container
-	if err := c.setupNetwork(vmi, vmi.Spec.Networks); err != nil {
+	if err := c.setupNetwork(vmi, netsetup.FilterNetsForMigrationTarget(vmi)); err != nil {
```

Comment:
I think this hook seem redundant because the is no logic and the VMI networks are given as is. 
Please consider removing this change.

Reply:
There is no logic at the moment, but I think we should plan for the future.

Reply:
We usually avoid having future preparations like these.

Reply:
The future is very close.
I'm working on a follow-up to abstract away the network setup from virt-handler's VM controller.

Reply:
Great, then maybe it make more sense to introduce this specific change in the planned follow-up PR, where it may have more context that demonstrate why its necessary.

Reply:
@ormergi ,
As I see it, this is handled at the same abstraction level.
We have 3 cases, 2 have specific logic and one is transparent. How can you work on the same abstraction level and express that for migration, we do not filter the networks? You are suggesting to use a comment?

Reply:
I suggest to not wrap it with a function that does nothing, comment can work.

Not sure about the incoming changes Orel mentioned though, if the follow-up PR would require root maintainer then maybe it make sense to have this function temporarily.